bin/scripts/org_migrator/org_migrator.py

```diff
@@ -211,7 +211,7 @@ def fetch_product_uuids(db, org_id):
         'UNION ' + \
         'SELECT pool.product_uuid FROM cp_pool pool WHERE pool.owner_id = %s ' + \
         'UNION ' + \
-        'SELECT akp.product_uuid FROM cp_activation_key ak JOIN cp2_activation_key_products akp ON akp.key_id = ak.id WHERE ak.owner_id = %s'
+        'SELECT akp.product_id FROM cp_activation_key ak JOIN cp_activation_key_products akp ON akp.key_id = ak.id WHERE ak.owner_id = %s'
```

Comment:
Like the content lookup, this is to ensure we have all the referenced products by UUID. Injecting product IDs into the mix won't work. This needs to be removed, and, potentially, changed to a lookup that fetches the product by its ID instead.